### PR TITLE
Remove Batches model. Just batch name in shred is enough.

### PR DESCRIPTION
**Pull request by xa4a**
As the title says. "Batches" seems to be unnecessary entity at the moment.

[Review on Reviewable](https://reviewable.io/reviews/dchaplinsky/unshred-tag/68)

### CODE REVIEW

File: cli/load_to_mongo.py [hunk 1]
```diff
@@ -184,7 +175,13 @@ def import_tags(drop=False):
 
 
 def list_batches():
-    return Batches.objects.order_by("created")
+    batch_counts = Shreds.objects.map_reduce(
+        map_f='function() { emit(this.batch, 1); }',
```
Comment (dchaplinsky) [inline cli/load_to_mongo.py]: :-\

**Comment by dchaplinsky** [comment]
What is the point in general. You've removed an entity, replaced it with a workaround, that actually increased the code complexity.

Where is the advantage?

**Comment by xa4a** [comment]
Batch instances are only used in a single place – as a CLI command, which is not the central part of the code. Having a model for it and storing it in the db just distracts anyone, who's trying to get the whole picture in his head.

File: cli/load_to_mongo.py (hunk 1 above)
Comment (dchaplinsky) [inline cli/load_to_mongo.py]: http://docs.mongodb.org/manual/reference/operator/aggregation/group/#group-by-month-day-and-year

Probably combined with: http://docs.mongoengine.org/apireference.html?highlight=aggregate#mongoengine.queryset.QuerySet.aggregate

**Comment by dchaplinsky** [comment]
One should get a bigger head?

Anyway, after some thinking I'm okay to drop it for now. Just get rid of ugly map_reduce part.

File: cli/load_to_mongo.py (hunk 1 above)
Comment (xa4a) [inline cli/load_to_mongo.py]: Here you go.

**Pull request merged**